rfc/rfc-51/rfc-51.md

```diff
@@ -148,20 +152,46 @@ hudi_cdc_table/
 
 Under a partition directory, the `.log` file with `CDCBlock` above will keep the changing data we have to materialize.
 
-There is an option to control what data is written to `CDCBlock`, that is `hoodie.table.cdc.supplemental.logging`. See the description of this config above.
+#### Persisting CDC in MOR: Write-on-indexing vs Write-on-compaction
+
+2 design choices on when to persist CDC in MOR tables:
+
+Write-on-indexing allows CDC info to be persisted at the earliest, however, in case of Flink writer or Bucket
```

Comment:
in case where row_key lookup is NOT done during writing (e.g. Bucket Index) the `op` data cannot be deduced and hence the CDC log block cannot be written out.

Reply:
So what is the solution here ?

Reply:
Ref: https://github.com/apache/hudi/pull/6256#discussion_r933909732